projects/dotcms-field-elements/src/components/dot-time/dot-time.e2e.ts

```diff
@@ -1,131 +1,399 @@
+                expect(spyStatusChangeEvent).toHaveReceivedEventDetail({
+                    name: '',
+                    status: {
+                        dotPristine: true,
+                        dotTouched: false,
+                        dotValid: true
+                    }
+                });
+                expect(spyValueChangeEvent).toHaveReceivedEventDetail({
+                    name: '',
+                    value: ''
+                });
+            });
+
+            it('should send status and value change and stop dot-input-calendar events', async () => {
+                const evt_statusChange = await page.spyOnEvent('_statusChange');
+                const evt_valueChange = await page.spyOnEvent('_valueChange');
+
+                await inputCalendar.triggerEvent('_valueChange', {
+                    detail: {
+                        name: '',
+                        value: '21:30:30'
+                    }
+                });
+                await inputCalendar.triggerEvent('_statusChange', {
+                    detail: {
+                        name: '',
+                        status: {
+                            dotPristine: false,
+                            dotTouched: true,
+                            dotValid: true
+                        }
+                    }
+                });
+                await page.waitForChanges();
+                expect(spyStatusChangeEvent).toHaveReceivedEventDetail({
+                    name: '',
+                    status: {
+                        dotPristine: false,
+                        dotTouched: true,
+                        dotValid: true
+                    }
+                });
+                expect(spyValueChangeEvent).toHaveReceivedEventDetail({
+                    name: '',
+                    value: '21:30:30'
+                });
+                expect(evt_statusChange.events).toEqual([]);
+                expect(evt_valueChange.events).toEqual([]);
+            });
+        });
+
+        describe('status change', () => {
+            it('should send status when dot-input-calendar send it', async () => {
+                await inputCalendar.triggerEvent('_statusChange', {
+                    detail: {
+                        name: '',
+                        status: {
+                            dotPristine: true,
+                            dotTouched: false,
+                            dotValid: false
+                        }
+                    }
+                });
+                await page.waitForChanges();
+                expect(spyStatusChangeEvent).toHaveReceivedEventDetail({
+                    name: '',
+                    status: {
+                        dotPristine: true,
+                        dotTouched: false,
+                        dotValid: false
+                    }
+                });
             });
-            expect(spyValueChange).toHaveReceivedEventDetail({ name: 'time01', value: '' });
```

Comment:
Missing the reset test methond AND I think the reset is missing resetting the status and values and emit those.

Reply:
is done here, https://github.com/dotCMS/core-web/pull/1049/files#diff-4b9e96b0cde2064489b698801412248eR320